src/AutorestSwift/Code Generator/SwiftGenerator.swift

```diff
@@ -63,6 +63,14 @@ class SwiftGenerator: CodeGenerator {
             andParams: [:]
         )
 
+        // Create Util.swift file
+        try render(
+            template: "UtilFile",
+            toSubfolder: .util,
+            withFilename: "Util",
+            andParams: [:]
+        )
+
```

Comment:
So I think the filename here should be DateUtil... or, if we can make it worth with an extension, "Date+Extensions"